ui/src/Logout.tsx

```diff
@@ -45,6 +45,4 @@ const mdtp = {
   resetFeatureFlags: reset,
 }
 
-export default ErrorHandling(
-  connect<{}, DispatchProps>(null, mdtp)(withRouter<WithRouterProps>(Logout))
-)
+export default connect<{}, DispatchProps>(null, mdtp)(withRouter(Logout))
```

Comment:
good call postponing the removal of `connect` for another PR. think we should audit the code for it later? or let it filter out naturally?

Reply:
for sure.  same for `withRouter`.